src/pr-info.ts

```diff
@@ -399,11 +398,17 @@ configSuspicious["OTHER_FILES.txt"] = contents =>
     // all path parts are not empty and not all dots
     : !contents.split(/\n/).every(line => line === "" || isRelativePath(line)) ? "bad path"
     : undefined;
-configSuspicious["tslint.json"] = contents =>
+configSuspicious["tslint.json"] = (contents, oldText) => {
     // just the required form
```

Comment:
The comment should get updated too: either the required form or deletions that get closer to it.

Reply:
Done :heavy_check_mark: